### ISSUE TITLE
docs: Vortex Configuration guide

### ISSUE THREAD

**Issue by AlessandroFinocchi** [revoltchat/vortex#23]
Hi, I'm having problems while configuring vortex. The Vortex program seems to work out but it's like it doesn't interface with Revolt: when I click on the button to connect with the voice chat it doesn't have any effect, but when I digit the Vortex URL it shows the following writing :

{"vortex":"0.3.0-alpha.1","features":{"rtp":true},"ws":"wss://vortex.revolt.chat"}

I have already followed every other guide in this site, so I'd like to ask for a more general guide in order to configure it. Thanks for the help.

**Comment by ghost** [comment]
> Hi, I'm having problems while configuring vortex. The Vortex program seems to work out but it's like it doesn't interface with Revolt: when I click on the button to connect with the voice chat it doesn't have any effect, but when I digit the Vortex URL it shows the following writing :
> 
> {"vortex":"0.3.0-alpha.1","features":{"rtp":true},"ws":"wss://vortex.revolt.chat"}
> 
> I have already followed every other guide in this site, so I'd like to ask for a more general guide in order to configure it. Thanks for the help.

I agree on this aspect i continually get the same issue, i check the debug console and no errors come threw i had to keep attempting till finally got the api working with vortex but when i click join voice call 💩 all happpens

**Comment by Fwuk** [comment]
> > Hi, I'm having problems while configuring vortex. The Vortex program seems to work out but it's like it doesn't interface with Revolt: when I click on the button to connect with the voice chat it doesn't have any effect, but when I digit the Vortex URL it shows the following writing :
> > {"vortex":"0.3.0-alpha.1","features":{"rtp":true},"ws":"wss://vortex.revolt.chat"}
> > I have already followed every other guide in this site, so I'd like to ask for a more general guide in order to configure it. Thanks for the help.
> 
> I agree on this aspect i continually get the same issue, i check the debug console and no errors come threw i had to keep attempting till finally got the api working with vortex but when i click join voice call 💩 all happpens

i also wish some sort of guide existed ive been trying to find out why i cant hear audio when i join a vc and ive tried to find the problem using chrome://webrtc-internals/ but i have had no luck. if anyone knows why that would be much appreciated.

**Comment by insertish** [comment]
Marking this issue for use later, Vortex will not be receiving any documentation in the near future since the developer behind it is currently not available. More over there's a lot of things which we want to rebuild or completely replace and we don't have the time spare to currently handle this.

I would recommend not self-hosting Vortex until it's received a proper overhaul, though as to consolidate discussion relating to the matter, I'm closing all related issues and pointing them to this comment (as to help out anyone looking for info anyways).

**Comment by cjeje84** [comment]
Hello, I have some errors, can you help me please?

Here is a screenshot and my config file

![image](https://user-images.githubusercontent.com/18289153/170141053-5e1f77c7-801c-4421-853a-fa3e0cb998a3.png)

Vortex config:
`HTTP_HOST=0.0.0.0:6080
WS_URL=ws://newdream.ml:9000
VOSO_WS_HOST=ws://newdream.ml:9000
MANAGE_TOKEN=moi

**Comment by Lon3lygoddx** [comment]
Hi guys i need help
   Compiling mediasoup-sys v0.3.3
error: failed to run custom build command for `mediasoup-sys v0.3.3`

Caused by:
  process didn't exit successfully: `/root/revolt/vortex/target/debug/build/mediasoup-sys-42397d80daadce11/build-script-build` (exit status: 101)
  --- stdout
  cargo:rustc-link-search=native=/usr/lib/gcc/x86_64-linux-gnu/10/
  cargo:rustc-link-lib=static=stdc++
  /root/revolt/vortex/target/debug/build/mediasoup-sys-885dd7fd8b1b84d9/out/out/pip/bin/meson setup \
        --prefix /root/revolt/vortex/target/debug/build/mediasoup-sys-885dd7fd8b1b84d9/out \
        --bindir '' \
        --libdir '' \
        --buildtype debug \
        -Db_pie=true \
        -Db_staticpic=true \
        --reconfigure \
        "" \
        /root/revolt/vortex/target/debug/build/mediasoup-sys-885dd7fd8b1b84d9/out/out/Debug/build || \
        /root/revolt/vortex/target/debug/build/mediasoup-sys-885dd7fd8b1b84d9/out/out/pip/bin/meson setup \
                --prefix /root/revolt/vortex/target/debug/build/mediasoup-sys-885dd7fd8b1b84d9/out \
                --bindir '' \
                --libdir '' \
                --buildtype debug \
                -Db_pie=true \
                -Db_staticpic=true \
                "" \
                /root/revolt/vortex/target/debug/build/mediasoup-sys-885dd7fd8b1b84d9/out/out/Debug/build
  The Meson build system


Help please...
